src/main/java/bio/terra/app/model/GoogleRegion.java

```diff
@@ -82,8 +88,10 @@ public static GoogleRegion fromValueWithDefault(String text) {
     }
 
     public static boolean matchingRegionWithFallbacks(List<StorageResource> storage, GoogleRegion region) {
-        for (StorageResource resource : storage) {
-            switch (resource.getCloudResource()) {
+        List<StorageResource> gcpStorage =
```

Comment:
I would use `var` here, and maybe use plural.
```suggestion
        var gcpStorageResources =
```

Reply:
whoa! I've never actually used var...time to start!